shoop_tests/core/test_order_creator.py

```diff
@@ -76,13 +80,16 @@ def test_order_creator_supplierless_product_line_conversion_should_fail(admin_us
         quantity=1,
         unit_price=TaxlessPrice(10),
     ))
-    creator = OrderCreator(request=None)
+
+    request = apply_request_middleware(RequestFactory().get("/"))
```

Comment:
In top-level test functions, you can use [the `rf` fixture](https://pytest-django.readthedocs.org/en/latest/helpers.html#rf-requestfactory) instead of having to import `RequestFactory` yourself:

```python
def test_order_creator(admin_user):
    request = blah(RequestFactory().blah)
```
-->
```python
def test_order_creator(admin_user, rf):
    request = blah(rf.blah)
```